internal/versions/versions.go

```diff
@@ -37,7 +37,7 @@ type Versions struct {
 type Handler interface {
 	restapi.VersionsAPI
 	GetMustGatherImages(openshiftVersion, cpuArchitecture, pullSecret string) (MustGatherVersion, error)
-	GetReleaseImage(openshiftVersion, cpuArchitecture string) (*models.ReleaseImage, error)
+	GetReleaseImage(openshiftVersion, cpuArchitecture string, strongMatch bool) (*models.ReleaseImage, error)
```

Comment:
This might be a time where we'd benefit from defining specific types for the matching strategies.
As is every time a see a call to this function I'm going to have to look at the definition to figure out what's going on.